src/AiFactory.cpp

```diff
@@ -439,26 +439,28 @@ void AiFactory::AddDefaultCombatStrategies(Player* player, PlayerbotAI* const fa
             engine->addStrategy("arena");
         }
 
-        engine->addStrategies("boost", "racials", "chat", "default", "aoe", "potions", "cast time", "dps assist", nullptr);
+        engine->addStrategies("boost", "racials", "chat", "default", "aoe", "potions", "cast time", "dps assist", "pvp", nullptr);
         engine->removeStrategy("custom::say");
         engine->removeStrategy("flee");
         engine->removeStrategy("threat");
-        engine->addStrategy("boost");
+        engine->removeStrategy("follow");
+        engine->removeStrategy("conserve mana");
+        engine->removeStrategy("cast time");
 
         if ((player->getClass() == CLASS_DRUID && tab == 2) || (player->getClass() == CLASS_SHAMAN && tab == 2))
             engine->addStrategies("caster", "caster aoe", nullptr);
 
         if (player->getClass() == CLASS_DRUID && tab == 1)
-            engine->addStrategies(/*"behind",*/ "dps", nullptr);
+            engine->addStrategies("behind", "dps", nullptr);
 
         if (player->getClass() == CLASS_ROGUE)
-            engine->addStrategies(/*"behind",*/ "stealth", nullptr);
+            engine->addStrategies("behind", "stealth", nullptr);
```

Comment:
Don’t just change it for granted, you need to make sure the strategies are working correctly and explain what specific problem they solve.